### DISCUSSION TITLE
HowTo doc question - How to setup servers, groups and SSH credentials

### DISCUSSION THREAD

**Discussion by Chaffy-0** [roxy-wi/roxy-wi#271] [Q&A] [answered]
I'm still new to the linux world, forgive me if this is a noob question regarding this first step.

First of all you have to create a local user on your HAProxy services. Let's call it a "proxyadmin". Generate SSH key for it or set a password. If you choose SSH key then upload it locally, use command ssh-copyid 127.0.0.1.

Is this referring to creating the local user on our current haproxy server or the haproxywi server called proxyadmin?  Does the haproxy service need to be running as this user in order for this to work correctly?

**Comment by Aidaho12** [comment, marked as answer]
User must be create on server there is you haproxy service. No, you have to change nothing for haproxy. Just create an user and give sudo to it
